tests/drivers/gpio/gpio_basic_api/boards/mimxrt1050_evk.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
```

Comment:
change copyright

Reply:
I kept the Nordic copyright because I copied the file from another board